tools/pom.xml

```
@@ -130,6 +130,24 @@
 				<version>1.2.0</version>
 			</dependency>
 
+			<dependency>
+				<groupId>javax.json.bind</groupId>
+				<artifactId>javax.json.bind-api</artifactId>
+				<version>1.0</version>
+			</dependency>
```

should this be scope provided?

---

```suggestion
			<dependency>
				<groupId>javax.json.bind</groupId>
				<artifactId>javax.json.bind-api</artifactId>
				<version>1.0</version>
				<scope>provided</scope>
			</dependency>
```

this should be provided by adding it as a dependency!

---

```suggestion
```

I don't think any pom changes are needed… just the server.xml, at least to build. Trying deploy..seemed like create snapshot worked with none of the pom changes